modules/battery_manager/php/testcontroller.class.inc

```diff
@@ -0,0 +1,211 @@
+<?php
+/**
+ * This file contains the Test Controller.
+ *
+ * PHP Version 7
+ *
+ * @category Module
+ * @package  Battery_Manager
+ * @author   Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\battery_manager;
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+
+/**
+ * Main class for managing Test Instances
+ *
+ * Handles requests for retrieving and saving Test Instances.
+ * Allows users to add, activate, and deactivate entries in the test battery.
+ *
+ * PHP Version 7
+ *
+ * @category Module
+ * @package  Battery_Manager
+ * @author   Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class TestController extends \NDB_Page
```

Comment:
I don't think this need to extend \NDB_Page. 
It could `implements RequestHandlerInterface` and it would be enough.

Reply:
I'm getting `404` errors when I try to do this.